site/learn/expert-advice/identity-basics/try-before-you-buy.md

```diff
@@ -0,0 +1,90 @@
+
+## Advantages of Trying Before You Buy
+
+Trying your authentication provider before you commit can save you lots of headaches in the future and should be an important part of choosing an authentication partner.
+
+Offering a free trial is an increasingly common practice among authentication providers, but if there's no trial option listed publicly, you should ask a salesperson if it's an option. If there still isn't a way to experiment with an auth system before you pick it, you may want to consider working with a different partner. There are many reasons you want to try before you commit, but let's go over a few of the more important ones.
+
+### Your Developers Can Test the API
+
+When making any significant purchase that your development team will use as part of their daily work, it's important to know that your developers can work with it and use it in ways that will benefit the product long term.
+
+If the developers start testing a potential API and they find they can build a relatively simple prototype to prove the API can support all your use cases, then you can be confident that a larger-scale implementation will work as well. Not only will this give you more confidence that the product you're buying is the right one for your business, it will also help your developers work more quickly when they do move to a larger-scale implementation because they will already have experience working with the API.
+
+For example, you may be looking at a few different authentication providers and have one that you've decided is the best candidate for your application. After all, it checks all the boxes! But when your developers start reading the documentation and actually build out a small prototype, they may find the library for your application's language doesn't support [passwordless authentication](/docs/v1/tech/guides/passwordless/). If you had committed to this authentication provider before discovering this, you might have already heavily integrated it into your application—which needs passwordless authentication.
+
+This would be a huge problem leading to a major rewrite or possibly having to drop a key feature of your app. However, if you had built a prototype using a free trial and discovered this limitation ahead of time, you could have simply moved on to choosing a different provider.
+
+### Better Understanding of Costs
+
+Many API-based products such as authentication providers are usage-based, which means as your usage increases, the pricing changes. While looking at the pricing table may have you convinced that you'll be on a free or low-cost plan, actually building out a prototype might reveal otherwise.
+
+There are all sorts of potential hidden costs or costs that you may not have considered from just thinking about your application at a high level. Knowing what this product costs on average for other companies of your scale is helpful.
```

Comment:
There are all sorts of potential hidden costs ~or costs~ that you may not have considered